### DISCUSSION TITLE
ONNX runtime using Blazor

### DISCUSSION THREAD

**Discussion by DFMERA** [microsoft/onnxruntime#14657] [Show & Tell]
Hi

I just tried Onnx runtime in a web app using Blazor and It works!!!
Here is the repo, may be you wanna share it in your social media!!

https://github.com/DFMERA/demo-ortblazor

![demo_ortblazor_gif](https://user-images.githubusercontent.com/23387464/218130944-ca1d34dd-5c12-4695-9490-ce09b4c0a25d.gif)

**Comment by natke** [comment]
Check this out @cassiebreviu

**Reply by cassiebreviu** [reply]
Yes, this is great! Currently it does work on the server-side inference for Balzor implementation but not the static web app WASM part of Blazor. The WASM onnxruntime file is currently not included and needs to be added as a native reference in csproj in order for the web assembly files to be packaged and sent to the browser for client-side inferencing. See this doc: https://learn.microsoft.com/en-us/aspnet/core/blazor/?view=aspnetcore-7.0#blazor-webassembly

**Reply by b-straub** [reply]
See https://github.com/microsoft/onnxruntime/issues/14962#issuecomment-1762148629
